modules/ROOT/pages/govern-api-instances.adoc

```diff
@@ -0,0 +1,39 @@
+= Governing API Instances
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+
+API Manager is closely integrated with xref:api-governance::index.adoc[Anypoint API Governance] to govern aspects of API instances with governance rulesets. Applying a governance ruleset to your API ensures that every instance of that API conforms to standards such as what policies are applied and if a TLS context is required.  
+
+To govern an API instance, see <<apply>>.
+
+After applying a governance ruleset to a specific API, view and fix the conformance issues for its API instances:
+
+* <<view-api-manager>>
+* <<view-api-governance>>
```

Comment:
This link is broken now since that section is removed. The organization here is a little odd too, because you send them to <<apply>> and then have an intro and other links. You could make this bullet points instead like:
To govern an API instance:
 * <<apply>>
* <<view-api-manager>>

Reply:
For some reason the text in the <>'s is invisible above, but it's the anchor names.